Slider/Assets/Scripts/Audio/AudioManager.cs

```diff
@@ -301,4 +320,76 @@ public static void SetSFXPitch(float value)
             s.source.pitch = s.pitch * value;
         }
     }
+
+    public static void EnqueueModifier(AudioModifier.ModifierType m)
+    {
+        if (_instance.modifiers.ContainsKey(m))
```

Comment:
if the vars are static then we can avoid trying to access `_instance` (caused issues elsewhere when testing)

i would prob make `EnqueueModifier()` and the others below static too